tmcprototype/CentralNode/CentralNode/CentralNode.py

```diff
@@ -407,6 +459,7 @@ def StartUpTelescope(self):
     @DebugIt()
     def AssignResources(self, argin):
         # PROTECTED REGION ID(CentralNode.AssignResources) ENABLED START #
+
```

Comment:
Why extra line break is added here?

Reply:
Removed it. The commit id is: https://github.com/ska-telescope/tmc-prototype/commit/dafa12fbac0e08db8ec3ec703afb99ba45db38c1